test/spec/ChoreoModelerSpec.js

```diff
@@ -118,9 +118,8 @@ describe('choreo modeler', function() {
 
     it('should export xml without error', function(done) {
       createModeler(choreoWithLoops).then(modeler => {
-        modeler.saveXML({ format: true }, function(err, xml) {
-          expect(err).to.be.undefined;
-        });
+        return modeler.saveXML({ format: true });
+      }).then(result => {
```

Comment:
I am not sure, but now this does not test anything anymore? There might be an exception if the export fails but there is no test. So I would either remove it or check result.